html-forms/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <div>
+      <div>
+        <label for="email-address">Email</label>
+      </div>
+      <input type="text" name="email" id="email-address" />
+    </div>
+
+    <div><label for="why-coding-cool">Why is coding cool?</label></div>
+    <select name="options" id="why-coding-cool">
+      <option value="">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+
+    <div>Favorite Language (choose one)</div>
+
+    <input
+      type="radio"
+      name="favoriteLanguage"
+      value="html"
+      checked="checked"
+      id="html" /><label for="html">HTML</label>
+    <input type="radio" name="favoriteLanguage" value="css" id="css" /><label
+      for="css"
+      >CSS</label
+    >
+    <input
+      type="radio"
+      name="favoriteLanguage"
+      value="javascript"
+      id="javascript" /><label for="javascript">JavaScript</label>
+
+    <div>
+      <label for="sign-up"
+        >Would you like to sign up for more information?</label
+      >
+    </div>
+    <input type="checkbox" name="signup" id="sign-up" /><label for="sign-up"
+      >Sign me up!</label
+    >
```

Comment:
Your indentation is really wacky here, which makes this hard to read. Please revise this to follow HTML formatting rules.